src/Entity.java

```diff
@@ -1,11 +1,8 @@
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-
-@JsonDeserialize(builder = EntityBuilder.class)
 public class Entity {
     private double health;
-    private final double attack;
+    private double attack;
     private final double defense;
```

Comment:
oh, a defense-re nem irtam, hogy nojon? my bad, de maradhat akkor.

Reply:
Ha jól tudom alapból nem is volt defense, csak hozzáadtuk ilyen "ha már van attack, akkor legyen defense" alapon. De ha akarod berakom azt is. :)